tests/realtikvtest/importintotest/import_into_test.go

```diff
@@ -146,14 +146,19 @@ func (s *mockGCSSuite) TestBasicImportInto() {
 	})
 	s.prepareAndUseDB("import_into")
 
-	// IMPORT INTO does not support some functions
 	s.tk.MustExec("drop table if exists t")
 	s.tk.MustExec("create table t (a bigint, b varchar(100), c int);")
 	s.tk.MustExec("set @v='test'")
-	sql := fmt.Sprintf(`IMPORT INTO t(a, @, c) SET b=tidb_is_ddl_owner() FROM 'gs://test-multi-load/db.tbl.001.csv?endpoint=%s'
+	// IMPORT INTO does not support column reference
+	sql := fmt.Sprintf(`IMPORT INTO t(a, @, c) SET b=a+1 FROM 'gs://test-multi-load/db.tbl.001.csv?endpoint=%s'
 		with thread=1`, gcsEndpoint)
-	err := s.tk.QueryToErr(sql)
-	require.ErrorContains(s.T(), err, "[0]can't use function at SET index 0")
+	err := s.tk.ExecToErr(sql)
+	require.ErrorContains(s.T(), err, "COLUMN reference is not supported in IMPORT INTO column assignment, index 0")
```

Comment:
This should be supported

Reply:
not now. I tried `load data`, mysql support it, but we don't